src/models/horizontal-scaling-summary/index.ts

```diff
@@ -14,4 +16,6 @@ export const HorizontalScalingSummaryModelValidationMap: PropTypes.ValidationMap
     maxReplicas: PropTypes.number.isRequired,
     currentCPUUtilizationPercentage: PropTypes.number.isRequired,
     targetCPUUtilizationPercentage: PropTypes.number.isRequired,
+    currentMemoryUtilizationPercentage: PropTypes.number.isRequired,
+    targetMemoryUtilizationPercentage: PropTypes.number.isRequired,
```

Comment:
if the API now may omit these values, they should be marked as optional
```suggestion
    currentCPUUtilizationPercentage: PropTypes.number,
    targetCPUUtilizationPercentage: PropTypes.number,
    currentMemoryUtilizationPercentage: PropTypes.number,
    targetMemoryUtilizationPercentage: PropTypes.number,
```